code/modules/cargo/packs.dm

```diff
@@ -530,6 +530,17 @@ var/list/all_supply_groups = list("Operations","Security","Hospitality","Enginee
 	containertype = /obj/structure/closet/crate
 	crate_name = "Mechanical maintenance crate"
 	group = "Engineering"
+	
+/datum/supply_pack/toolmods
```

Comment:
I'd like a toolmod to be worth about 250 credits, adjust the quantity or the price of the crate appropriately